core/src/data/value/into.rs

```diff
@@ -417,6 +417,10 @@ mod tests {
             Err(ValueError::ImpossibleCast.into())
         );
         test!(Value::Null, Err(ValueError::ImpossibleCast.into()));
+
+        //impossible casts to bool
```

Comment:
<https://doc.rust-lang.org/book/ch03-04-comments.html>
Similarly, in `GlueSQL` it is common to use a leading space like `// TEMP`.